app/controllers/collection_drafts_controller.rb

```diff
@@ -4,11 +4,13 @@ class CollectionDraftsController < BaseDraftsController
   include ControlledKeywords
   include CMRCollectionsHelper
   include GKRKeywordRecommendations
-  before_action :set_resource, only: [:show, :edit, :update, :destroy, :publish]
+  before_action :set_resource, only: [:download_json, :show, :edit, :update, :destroy, :publish]
   before_action :load_umm_c_schema, only: [:new, :edit, :show, :publish]
   before_action :collection_validation_setup, only: [:show, :publish]
   before_action :verify_valid_metadata, only: [:publish]
   before_action :set_associated_concepts, only: [:show]
+  before_action :ensure_user_is_logged_in, except: [:download_json]
```

Comment:
this should be inherited from ApplicationController, so all that is necessary is to `skip_before_action` for the method you're specifying

Reply:
Ah makes sense, will change.